schemas/common/descriptors/display/alternateDisplayInfo.schema.json

```diff
@@ -43,6 +43,16 @@
             }
           }
         },
+        "xdm:note": {
+          "title": "Note",
+          "type": "object",
+          "description": "When present, user friendly note to display. Similar to a description, but provides more details about the actual usage of the field.",
```

Comment:
Why is this a set of properties and not a simple string - esp. if it's to be "user friendly"??

Reply:
Like all the attributes on this descriptor, they support a localized value.
"xdm:note": {
   "en-us": "My English Note",
   "fr-fr": "My French note", 
}

Reply:
If that's the intent, then they all need to be changed such that the key is defined to match a pattern for ISO 3166 while the value is open.

Reply:
I thought the current "patternProperties" regular expression was doing that. If not, please provide the regular expression which does.